lib/slax/channels/channel.ex

```diff
@@ -0,0 +1,17 @@
+defmodule Slax.Channel do
+  use Slax.Schema
+
+  @type t :: %__MODULE__{}
+  schema "channels" do
+    field(:channel_id, :string)
+    field(:name, :string)
+    field(:disabled, :boolean, default: false)
+
+    timestamps()
+  end
+
+  def changeset(model, params \\ %{}) do
+    model
+    |> cast(params, [:id, :name, :disabled])
```

Comment:
not a big deal, but usually we don't use a pipeline unless there's more than one function call involved